terraform_create_images/modules/loader_vm/scripts/create_and_load_everything_from_scratch.sh

```diff
@@ -77,36 +76,36 @@ intermediateTables=(
   v2d_sa_molecular_trait
   l2g
   manhattan
-  v2g_scored
-  d2v2g_scored
 )
 ## Create intermediary tables
 for t in "${intermediateTables[@]}"; do
   echo "[Clickhouse] Creating intermediary table: ${t}_log"
-  clickhouse-client -m -n <"${SCRIPT_DIR}/${t}_log.sql"
+  clickhouse-client -h ${CLICKHOUSE_HOST} -m -n <"${SCRIPT_DIR}/clickhouse/sql/${t}_log.sql"
```

Comment:
The path is incorrect. We use this script in combination with Terraform and GCP. When the VM startup script is called it copies the sql files from Github to the VM under /tmp/scripts/ which is where this script expects to find them. To make this change work you'd also have to update `terraform_create_images/modules/loader_vm/scripts/vm-start-up-script.sh` to create a more complex directory structure.

Reply:
I removed the change to point to SQL files in sub-directories.